components/ProfileHead.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Link from "next/link";
+
+const ProfileHead = () => {
+  return (
+    <div className="bg-white max-w-2xl overflow-hidden flex flex-row m-auto mb-4 p-4 border border-gray-900">
+      <div
+        className="w-16 h-16 bg-pink-400 rounded-full mr-4"
+        style={{
+          backgroundImage: "url('https://thispersondoesnotexist.com/image')",
+          backgroundRepeat: "no-repeat",
+          backgroundPosition: "center",
+          backgroundSize: "cover",
+        }}
+      />
+      <div className="flex flex-col">
+        <div className="flex flex-row items-center">
+          <p className="text-xl mr-2">User Name</p>
+          <p className="text-green-500">
+            <Link href="#">Follow</Link>
```

Comment:
Why is there a follow button if this is our own profile?

Reply:
I put that there as a placeholder since i figure we can reuse this header for all profiles - perhaps the `follow` button could be an `edit profile` button if profile === self

Reply:
or it could just hide all together if it's your own profile

Reply:
Sounds good to me!